aries-site/src/pages/templates/app-layouts.mdx

```diff
@@ -0,0 +1,170 @@
+screen size or resolution.
+
+## Page Container
+The Page Container is responsible for the layout for the main content of a page.
+It contains a page's [Page Header](/components/header#header-for-a-single-page) 
+and its Page Content.
+
+### Container Width
+There are 3 content width options from which to choose, depending on 
+the layout use case: [Wide](#wide-default), [Narrow](#narrow) and [Full](#full). 
+
+In all cases, the minimum width of content is `336px`, after which horizontal 
+scrolling is enabled.
+
+#### Wide (default)
+This is the default width for the Page Container. Using the "wide" content width 
+scales the Page Container until a maximum width of `1608px`. If the screen is 
+larger than `1608px`, the Page Container will be center aligned and white space 
+will be displayed around the perimeter of the content. 
+
+--INSERT-EXAMPLE-HERE--
+
+Use cases include:
+- Dashboards 
+- Tables 
+- Lists
+- Charts
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+#### Narrow
+The "narrow" option scales the content width until a maximum width of `768px`. 
+If the screen is larger than 768, the Page Container will be center aligned 
+and white space will be displayed around the perimeter of the content. 
+
+--INSERT-EXAMPLE-HERE--
+
+Use cases include:
+- Wizards 
+- Articles 
+- Single task flows, such as forms
+- Admin and user settings
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+#### Full
+This option will automatically resize the container to fill the 
+screen displaying it. With the "full" option, if the screen grows wider 
+than the available content, the Page Container will be left aligned to 
+the edge of the screen.
+
+--INSERT-EXAMPLE-HERE--
+
+Use cases include:
+- Charts 
+- Maps 
+- Tables
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+--INSERT-EXAMPLE-HERE-?????-TBD--
+
+### Grid
+Grommet uses a combination of CSS Grid and Flexbox. CSS Grid 
+allows for the creation of custom grids depending on specific 
+content needs. Items on smaller screens no longer have to stack 
+in the order they are arranged on desktop (unless that is the 
+desired output). 
+
+#### Ordering items based on content needs
+Focusing on content needs can help us create better optimal user 
+experiences &mdash; think about which piece of content a user is likely 
+to interact with first and order items on the page accordingly. 
+
+Specifying layout content in this way will also ensure that it 
+is interepreted appropriately by screen readers for visually 
+impaired users.
+
+--INSERT-EXAMPLE-HERE--
+
+#### Grid Spacing
+Grid allows you to specify row and columns sizes, as well as define 
+margin and gap sizing. The HPE Design System components and templates 
+have grid sizing built in, but when you need them to craft your digital 
+experience, use these values as a reference:
+
+**Row and Column Sizes**  
+
+--PRESENT-THE-FOLLOWING-AS-A-TABLE-OR-MAYBE-EVEN-A-NAMEVALUELIST--
```

Comment:
Im thinking a `table` to display these values

Reply:
done.